### PR TITLE
Add .kar extension as known type

### PR DESCRIPTION
**Pull request by cstamas**

### CODE REVIEW

**Comment by mrprescott** [comment]
+1

**Comment by mcculls** [comment]
+1

File: components/nexus-mime/src/main/resources/builtin-mimetypes.properties [hunk 1]
```diff
@@ -30,3 +30,4 @@ tgz = application/x-tgz,application/x-gzip,application/gzip
 bz2 = application/x-bzip2
 tbz = application/x-bzip2,application/x-bzip
 nupkg = application/zip
+kar = application/zip,application/x-zip-compressed
```
Comment (mcculls) [inline components/nexus-mime/src/main/resources/builtin-mimetypes.properties]: btw, any particular reason why this entry also has `x-zip-compressed` but nupkg doesn't?

Comment (cstamas) [inline components/nexus-mime/src/main/resources/builtin-mimetypes.properties]: No reason, I just went the "safe side" and added all the Tike mime magic detection detected (as seen on DEBUG level of repository logger). But basically it's the leftmost that is considered the "typical" type from this properties file.

**Comment by mcculls** [comment]
Manually merged as 172628c

**Pull request closed without merging**